src/components/activity/ActivityIntroductionForm.tsx

```diff
@@ -34,6 +35,7 @@ const ActivityIntroductionForm: React.FC<{
     hasura.UPDATE_ACTIVITY_INTRODUCTIONVariables
   >(UPDATE_ACTIVITY_INTRODUCTION)
   const [loading, setLoading] = useState(false)
+  const [coverId, setCoverId] = useState(uuid())
```

Comment:
i think we may not need to use the state here. how about just use `const coverId = uuid()` directly?

Reply:
fixed